README.md

```diff
@@ -94,6 +94,7 @@ Example   | Description |
 [Twitter](aws-ts-twitter-athena) | Query Twitter every 2 minutes, store the results in S3, and set up an Athena table and query.
 [URL Shortener](aws-ts-url-shortener-cache-http) | Create a serverless URL shortener that uses high-level components.
 [Voting App](aws-ts-voting-app) | Create a simple voting app using Redis and Python Flask.
+[Web Server](aws-ts-webserver) | Deploy an EC2 Virtual machine running a Python web server.
```

Comment:
```suggestion
[Web Server](aws-ts-webserver) | Deploy an EC2 Virtual machine using TypeScript to run a Python web server.
```

Reply:
Not sure if the suggestion above is necessary, but given we're mixing languages, it might be useful to clarify.